modules/dop/endpoints/configmanage.go

```diff
@@ -347,12 +352,14 @@ func (e *Endpoints) generatorPipelineNS(appID uint64) (*apistructs.PipelineConfi
 			return nil, apierrors.ErrFetchConfigNamespace.InternalError(err)
 		}
 		if branchPrefix == gitflowutil.HOTFIX_WITHOUT_SLASH || branchPrefix == gitflowutil.SUPPORT_WITHOUT_SLASH {
-			ns := fmt.Sprintf("%s-%d-%s", apistructs.PipelineAppConfigNameSpacePreFix, appID, branchPrefix)
-			configs, err := e.bdl.GetPipelineCmsNsConfigs(ns, apistructs.PipelineCmsGetConfigsRequest{
-				PipelineSource: "dice",
-			})
+			ns := fmt.Sprintf("%s-%d-%s", cms.PipelineAppConfigNameSpacePrefix, appID, branchPrefix)
+			configs, err := e.pipelineCms.GetCmsNsConfigs(utils.WithInternalClientContext(context.Background()),
+				&cmspb.CmsNsConfigsGetRequest{
+					Ns:             ns,
+					PipelineSource: "dice",
```

Comment:
this hard-coded "dice" should be const or refactor it out.

if cannot refactor at this time, comment TODO

Reply:
Adjusted it by the way.